erpnext/public/js/utils/serial_no_batch_selector.js

```diff
@@ -313,7 +315,21 @@ erpnext.SerialNoBatchSelector = Class.extend({
 
 		qty_field.set_input(total_qty);
 	},
+	update_pending_qtys: function() {
+		const pendingQtyField = this.dialog.fields_dict.pending_qty;
```

Comment:
We don't use camel casing in variable names, please rename all variables using "_" like `pending_qty_field`